.github/workflows/prepare-transaction.yml

```
@@ -14,13 +14,14 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
-        with:
-          sparse-checkout: |
-            .github/scripts/validate_new_chain_request.sh
-          sparse-checkout-cone-mode: false
-
       - name: Run the validation script
-        run: bash ./.github/scripts/validate_new_chain_request.sh
+        run: |
+          RPC_URL=$(echo "$ISSUE_BODY" | grep -E -o 'https?://[^ ]+' -m 1 | head -1)
+          SUMMARY_FILE=summary.json RPC=$RPC_URL npm run verify:new-chain-request
+          RESPONSE=$(jq -r '.response' $SUMMARY_FILE)
+          echo "COMMENT_OUTPUT=$RESPONSE" >> $GITHUB_ENV
+          LABEL_OPERATION=$(jq -r '.labelOperation' $SUMMARY_FILE)
+          echo "LABEL_OPERATION=$LABEL_OPERATION" >> $GITHUB_ENV
```

Personally, style-wise, I find the following more succinct and the former a bit repetitive (conceptually, we set the variable, and then set the variable again right after - instead of only setting it once).

```suggestion
          echo "COMMENT_OUTPUT=$(jq -r '.response' $SUMMARY_FILE)" >> $GITHUB_ENV
          echo "LABEL_OPERATION=$(jq -r '.labelOperation' $SUMMARY_FILE)" >> $GITHUB_ENV
```